tests/test_project/admin.py

```diff
@@ -28,9 +29,16 @@ class OperatorInline(admin.StackedInline):
 
 
 @admin.register(Project)
-class ProjectAdmin(admin.ModelAdmin):
-    inlines = [OperatorInline]
-    list_display = ['name']
+class ProjectAdmin(UUIDAdmin, ReceiveUrlAdmin):
+    receive_url_name = 'receive_project'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.inlines = [OperatorInline]
+        self.list_display = ('name',)
+        self.fields = ('uuid', 'name', 'key', 'receive_url',)
+        self.readonly_fields = ('uuid', 'receive_url',)
```

Comment:
why you can't define these attributes as class attributes instead of having to define an `__init__` method for it?